neofetch

```diff
@@ -511,6 +512,10 @@ getdistro () {
             distro="${distro/Caption'='}"
             distro="${distro/Microsoft }"
         ;;
+
+        "Solaris")
+            distro="$(cat /etc/release | head -1 | awk ' {print $1 " " $2} ')"
```

Comment:
You can probably remove both pipes here with a line similar to linux's /etc/release part.

```sh
awk -F 'NAME=' '/^NAME=/ {printf $2}' /etc/*ease
```